### PR TITLE
Initialize package for storybook full source code babel preset

### PR DESCRIPTION
**Pull request by ling1726**
Preparing to move storybook codesandbox code to monorepo

Addresses #26780

### CODE REVIEW

**Review by Hotell — APPROVED** [review]

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/0a3196e08cfeba3068f4100e0faff2771a24fcdc'>0a3196e08cfeba3068f4100e0faff2771a24fcdc</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26781/perf-test-react-components/Avatar-mount_base.html">1347</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26781/perf-test-react-components/Avatar-mount.html">1359</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26781/perf-test-react-components/Button-mount_base.html">967</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26781/perf-test-react-components/Button-mount.html">954</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26781/perf-test-react-components/Field-mount_base.html">2061</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26781/perf-test-react-components/Field-mount.html">2076</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26781/perf-test-react-components/FluentProvider-mount_base.html">1696</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26781/perf-test-react-components/FluentProvider-mount.html">1688</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26781/perf-test-react-components/FluentProviderWithTheme-mount_base.html">634</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26781/perf-test-react-components/FluentProviderWithTheme-mount.html">661</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26781/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">601</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26781/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">617</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26781/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">646</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26781/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">637</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26781/perf-test-react-components/InfoButton-mount_base.html">582</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26781/perf-test-react-components/InfoButton-mount.html">572</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26781/perf-test-react-components/MakeStyles-mount_base.html">1947</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26781/perf-test-react-components/MakeStyles-mount.html">1917</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26781/perf-test-react-components/Persona-mount_base.html">3065</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26781/perf-test-react-components/Persona-mount.html">3092</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26781/perf-test-react-components/SpinButton-mount_base.html">2578</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26781/perf-test-react-components/SpinButton-mount.html">2609</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
:warning: No perf measurements available
<div id="perfComment9424"/>

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -211,6 +211,7 @@ packages/react-components/react-datepicker @microsoft/cxe-red @sopranopillow @kh
 packages/react-components/react-migration-v8-v9 @microsoft/cxe-red @microsoft/cxe-coastal @geoffcoxmsft
```
Comment (fabricteam) [inline .github/CODEOWNERS:211]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26781&commitId=74c97cc48d602045437b317742e75c0b2cfeca01&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 1043 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUI-v0 No visual regressions between this PR and main
<div id="vrtCommentFluentUI-v0"/>

**Review by JustSlone — APPROVED** [review]

**Pull request merged**